src/java/org/apache/cassandra/io/util/PathUtils.java

```diff
@@ -334,6 +336,52 @@ private static void deleteRecursive(Path path, RateLimiter rateLimiter, Consumer
         delete(path, rateLimiter);
     }
 
+    /**
+     * Recursively delete the content of the directory, but not the directory itself.
+     */
+    public static void deleteContent(Path path)
+    {
+        deleteContent(new File(path));
+    }
```

Comment:
I believe this function should be implemented through `deleteRecursive` and in a similar fashion. E.g.:
```suggestion
    public static void deleteContent(Path path)
    {
        if (isDirectory(path))
            forEach(path, PathUtils::deleteRecursive);
    }
```

Reply:
Agree, thanks, I should have noticed that.